### PR TITLE
Increase coverage of Slicey

### PR DESCRIPTION
**Pull request by camjc**

### CODE REVIEW

File: test/components/alexandria/slicey/ArcComponentTest.js [hunk 1]
```diff
@@ -6,13 +6,27 @@ import createComponent from 'helpers/shallowRenderHelper';
...
+  it('should render an arc for given data', () => {
+    const props = {
+      data: {
+        label: "Something Great",
+        id: 0,
+        largeArcFlag: 0,
+        x1: 3.06,
+        y1: -0.5,
+        x2: 0.14,
+        y2: 0.47,
```
Comment (mcslash) [inline test/components/alexandria/slicey/ArcComponentTest.js]: extra comma?

File: test/components/alexandria/slicey/DataProcessorTest.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+/* eslint-env node, mocha */
+/* global expect */
+/* eslint no-console: 0*/
```
Comment (mcslash) [inline test/components/alexandria/slicey/DataProcessorTest.js]: being pedantic, but space after 0.

**Comment by mcslash** [comment]
schweeet! :candy:

File: test/components/alexandria/slicey/DataProcessorTest.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+import DataProcessor from 'components/alexandria/slicey/DataProcessor.js';
```
Comment (tchcxp) [inline test/components/alexandria/slicey/DataProcessorTest.js]: no need for `.js`

**Comment by tchcxp** [comment]
Looking really good. :)

File: test/components/alexandria/slicey/DataProcessorTest.js (hunk 2 above)
Comment (camjc) [inline test/components/alexandria/slicey/DataProcessorTest.js]: that's supposed to be there.

File: test/components/alexandria/slicey/ArcComponentTest.js (hunk 1 above)
Comment (camjc) [inline test/components/alexandria/slicey/ArcComponentTest.js]: nope. See any other PR for explanation.

**Comment by mcslash** [comment]
Bingo bango merging this sucker! :sunglasses:

**Pull request merged**